src/component/app/index.tsx

```diff
@@ -1,37 +1,36 @@
+import { useAuth0 } from "@auth0/auth0-react";
```

Comment:
Using `JS/TS Import/Export Sorter` VS Code Extension to keep these alphabetically ordered and grouped by external then local imports (like `goimports`).